src/scripts/recon/allowed-methods/__main__.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
```

Comment:
Can you, please, rename the directory name from `/allowed-methods` to `/allowed_methods` (with an underscore)? To avoid some possible problems with imports in future (if the structure will be changed). Thanks!